### DISCUSSION TITLE
连接桩连线为非直线时无法使用tooltip

### DISCUSSION THREAD

**Discussion by usernamewt** [antvis/X6#2571] [Q&A]
![image](https://user-images.githubusercontent.com/50309383/186388325-8aaf49fa-cbf1-4ec9-80a0-2808c9c3b52c.png)
![image](https://user-images.githubusercontent.com/50309383/186389108-02f18205-2dc5-4712-9b77-3e265ef5a76c.png)

**Comment by zmcode** [comment]
展示你的代码
